### PR TITLE
RELEASE-616: PLF 4.4.5

### PR DESCRIPTION
**Pull request by mhbensghaier**

### CODE REVIEW

File: exo-platform/RELEASE-616.json [hunk 1]
```diff
@@ -0,0 +1,839 @@
...
+      "git_organization":"exoplatform",
+      "labels":"addons,trial",
+      "maven_property_version":"addon.exo.video.calls.version",
+      "container": {
+        "image": "exoplatform/release:jdk8-maven32",
+        "step": 1
+      },
+      "release":{
+        "branch":"stable/1.4.x",
+        "version":"1.4.4",
```
Comment (vsellier) [inline exo-platform/RELEASE-616.json]: should be 1.4.5-RC01

File: exo-platform/RELEASE-616.json [hunk 2]
```diff
@@ -0,0 +1,839 @@
...
+      "git_organization":"exoplatform",
+      "labels":"exo",
+      "maven_property_version":"org.exoplatform.platform.addons-manager.version",
+      "container": {
+        "image": "exoplatform/release:jdk7-maven32",
+        "step": 1
+      },
+      "release":{
+        "branch": "develop",
+        "version": "1.1.4",
```
Comment (vsellier) [inline exo-platform/RELEASE-616.json]: Should be 1.1.5-RC01

File: exo-platform/RELEASE-616.json [hunk 3]
```diff
@@ -0,0 +1,839 @@
...
+      "git_organization":"exoplatform",
+      "labels":"addons,trial",
+      "maven_property_version":"addon.exo.chat.version",
+      "container": {
+        "image": "exoplatform/release:jdk8-maven32",
+        "step": 1
+      },
+      "release":{
+        "branch":"stable/1.5.x",
+        "version":"1.5.1",
```
Comment (vsellier) [inline exo-platform/RELEASE-616.json]: should be 1.5.2-RC01

File: exo-platform/RELEASE-616.json [hunk 4]
```diff
@@ -0,0 +1,839 @@
...
+      "git_organization":"exoplatform",
+      "labels":"addons",
+      "maven_property_version":"addon.exo.cmis.version",
+      "container": {
+        "image": "exoplatform/release:jdk8-maven32",
+        "step": 1
+      },
+      "release":{
+        "branch":"stable/4.4.x",
+        "version":"4.4.5",
```
Comment (vsellier) [inline exo-platform/RELEASE-616.json]: 4.4.5-RC01

File: exo-platform/RELEASE-616.json [hunk 5]
```diff
@@ -0,0 +1,839 @@
...
+      "git_organization":"exoplatform",
+      "labels":"addons",
+      "maven_property_version":"addon.exo.crash.version",
+      "container": {
+        "image": "exoplatform/release:jdk8-maven32",
+        "step": 1
+      },
+      "release":{
+        "branch":"stable/4.4.x",
+        "version":"4.4.5",
```
Comment (vsellier) [inline exo-platform/RELEASE-616.json]: 4.4.5-RC01

**Comment by mhbensghaier** [comment]
change done

**Pull request merged**